safe-room.lic

```diff
@@ -102,6 +102,9 @@ class SafeRoom
       break if result =~ /^What were you/
       cast_spell(settings.necromancer_healing['Devour'], settings)
       devours -= 1 unless right_hand
+      while DRSpells.active_spells['Devour']
+        pause 5
+      end
     end
```

Comment:
Do we prevent a race condition with lag by using `pause 5 until !DRSpells.active_spells['Devour']`?

Reply:
that is definitely cleaner :)